cmd/insolar/main.go

```diff
@@ -116,6 +117,90 @@ func parseInputParams() {
 
 }
 
+func main() {
+	if u := os.Getenv("INSOLAR_API_URL"); u != "" {
```

Comment:
What's the reason for creating env variable? It's simply configured in the cli using arguments